scripts/pull_request_blaster_outer/change_travis_ruby_versions.rb

```diff
@@ -12,13 +12,15 @@
   exit 0
 end
 
-require 'yaml'
-yml = YAML.load_file(travis)
-unless yml.key?("rvm")
-  puts "#{travis} doesn't have rvm key, skipping"
-  exit 0
-end
+travis_content = File.read(travis)
 
-yml["rvm"] = versions
+changed = false
+versions.each do |version|
+  major_minor = version[0,3]
```

Comment:
this will break in ruby 3.11 :trollface: 
but, still, maybe a split?

Reply:
haha, yup, good catch